posteriors/laplace/dense_fisher.py

```diff
@@ -130,9 +128,8 @@ def update(
         )(state.params)
 
     if inplace:
-        state.prec += fisher
-        state.aux = aux
-        return state
+        state.prec.data += fisher
```

Comment:
iiuc `state.prec.data` and `_replace` have to be used because NamedTuple is immutable. Imo this should be explained somewhere (eg in getting started)

Reply:
Good point! I'll expand the `inplace` gotchas section to discuss how this is handled in the code